recipes/cyrus-sasl/meta.yaml

```diff
@@ -11,25 +11,30 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: ftp://ftp.cyrusimap.org/cyrus-sasl/{{ name }}-{{ version }}.tar.gz
   {{ hashtype }}: {{ hash }}
+  patches:
+    - makefile.patch  # [osx]
+    - include_makefile.patch  # [osx]
+    - lib_makefile.patch  # [osx]
 
 build:
-  number: 1
+  number: 0
   script:
-    - ./configure --prefix=$PREFIX
+    - ./configure --prefix=$PREFIX  # [linux]
+    - ./configure --prefix=$PREFIX --disable-macos-framework  # [osx]
```

Comment:
Do we still need all the patches with this option?

Another thing one could try is overriding `framedir` to point to `$PREFIX/Frameworks`. IDK if we want to start packaging frameworks, but then it would be in one place where we could make that decision or delete it.

Reply:
Just noting because patches tend to be more maintenance burden compared to build options or build steps that we control. Something to think about in the feedstock.

Reply:
Certainly.  I think in general right now we don't package frameworks.
To change the framedir to point somewhere else is still going to require patches though due to the way they've written their makefiles.